docs/source/plugins.rst

```diff
@@ -137,7 +137,9 @@ The following methods are options to connect the plugin to pygeoapi:
          data: /path/to/file
          id_field: stn_id
 
-BEGIN
+
+.. note::  The United States Geological Survey has created a Cookiecutter project for creating pygeoapi plugins. See the `pygeoapi-plugin-cookiecutter`_ project to get started.
```

Comment:
@dblodgett-usgs let me know if this is okay or if the wording needs to be updated.

Reply:
Perfect.